core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerDataWriter.java

```diff
@@ -69,8 +70,8 @@ public static BlockWorkerDataWriter create(final FileSystemContext context, long
     Preconditions.checkNotNull(blockWorker, "blockWorker");
     long sessionId = SessionIdUtils.createSessionId();
     try {
-      blockWorker.createBlock(sessionId, blockId, options.getWriteTier(), options.getMediumType(),
-          reservedBytes);
+      blockWorker.createBlock(sessionId, blockId, options.getWriteTier(),
+          new CreateBlockOptions("", options.getMediumType(), reservedBytes));
```

Comment:
Can you help add  @Nullable to the alluxiopath and medium input?
```
  public CreateBlockOptions(String alluxioPath, String medium, long initialBytes) {
```
null is the default or "" is the default?

Reply:
added